app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -89,28 +89,67 @@
     private int offset = 1;
     private ChatFeedRecyclerAdapter recyclerAdapter;
     private Realm realm;
-    private TextView txtSpeechInput;
-    private ImageButton btnSpeak;
+    private TextView edittext;
+    private ImageButton btn;
     private SharedPreferences shre;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
         init();
+        btn = (ImageButton) findViewById(R.id.btnSpeak);
```

Comment:
@jig08 I had some suggestions on this PR that I was going to type.
Please assign yourself before you are going to merge a Pull Request. :slightly_smiling_face: 
@mayank408 We are already using Butterknife for View Binding so use it instead of the conventional method of findviewbyid.